recipes/nvmolkit/recipe.yaml

```diff
@@ -0,0 +1,67 @@
+context:
+  version: "0.2.0"
+
+package:
+  name: "nvmolkit"
+  version: ${{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/nvMolKit/archive/v${{ version }}.tar.gz
+  sha256: 8c0181c4dcd02aac42dc3c1b783ac677a8149fd5d5b5d25e85ba30e85bd6ec5a
+
+build:
+  number: 0
+  skip:
+    - osx
+    - win
+    - cuda_compiler == 'None'
+requirements:
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('cuda') }}
+    - cmake >=3.30
+    - make
+  host:
+    - cuda-runtime
+    - cuda-version >=12.6
+    - eigen
+    - libboost-devel
+    - libboost-headers
+    - libboost-python-devel
+    - cuda-libraries-dev
+    - cuda-nvtx-dev
+    - libgomp
+    - librdkit-dev
+    - pip
+    - python
+    - rdkit ==${{ rdkit }}
+    - setuptools
+    - scikit-build
+  run:
+    - cuda-runtime
+    - cuda-version >=12.6
```

Comment:
The CUDA Compiler [sets a runtime dependency on `cuda-version` with an appropriate version constraint]( https://github.com/conda-forge/cuda-nvcc-feedstock/blob/23f27c97cab74fac04fb0f2cedc6b701020075e8/recipe/meta.yaml#L63 )

So the proper constraint is set automatically taking into account the CUDA version we built against

Meaning we can just drop this

```suggestion
```